src/SixLabors.Fonts/Tables/General/CMapTable.cs

```diff
@@ -13,46 +13,29 @@ namespace SixLabors.Fonts.Tables.General
     internal sealed class CMapTable : Table
     {
         internal const string TableName = "cmap";
-        private readonly CMapSubTable[] platformTables;
 
-        public CMapTable(CMapSubTable[] tables)
+        private static readonly Dictionary<PlatformIDs, int> PreferredPlatformOrder = new Dictionary<PlatformIDs, int>
```

Comment:
On my phone but isn’t this already an enum? A simple cast will be much faster than Dictionary + TryGet

Reply:
Indeed it's an enum but the `PreferredPlatformOrder` dictionary defines the preferred *order* of the platform ids, i.e. Windows first to preserve the semantics of the previous code.

Reply:
Ah yep, I see now, nice!